src/wanderer/lib.ts

```diff
@@ -83,8 +83,13 @@ export function unlock(loc: Location, value: number): boolean {
 }
 
 const backupSkiplist = $locations`The Overgrown Lot, The Skeleton Store, The Mansion of Dr. Weirdeaux`;
+// These are locations where all non-combats have skips or lead to a combat.
+const backupSafelist = $locations`The Haunted Gallery, The Haunted Ballroom, The Haunted Library, The Penultimate Fantasy Airship, Cobb's Knob Barracks, The Castle in the Clouds in the Sky (Basement), The Castle in the Clouds in the Sky (Ground Floor), The Castle in the Clouds in the Sky (Top Floor), The Haiku Dungeon, Twin Peak, A Mob of Zeppelin Protesters`;
```

Comment:
It might be worth breaking up the "have skips" and the "or lead to a combat" chunks here--the former is also safe to use for YRs but the latter is not

Reply:
The YR wandering should also be safe for the ones that lead to a combat, no? From what I understand those monsters are also included in the location monsters, or am I mistaken? This is what lead to YR's being used in the ballroom for a while iirc

Reply:
For YRs, at the very least, we do need to treat them differently. We use `appearanceRates` now, which should filter out things like the ballroom guy